components/verify-email-screen.js

```diff
@@ -10,25 +13,55 @@ const _handleClick = () => window.location.reload()
  * (One way to make sure the parent page fetches the latest email verification status
  * is to simply reload the page.)
  */
-const VerifyEmailScreen = () => (
-  <div>
-    <h1>Verify your email address</h1>
-    <p>
-      Please check your email inbox and follow the link in the message
-      to verify your email address before finishing your account setup.
-    </p>
-    <p>
-      Once you're verified, click the button below to continue.
-    </p>
+class VerifyEmailScreen extends Component {
+  resendVerificationEmail = () => {
+    const { accessToken } = this.props.auth
+    if (!accessToken) {
+      console.warn('No access token found.')
+      return
+    }
+    secureFetch(`${API_USER_URL}/verification-email`, accessToken)
+      .then(json => window.alert('Verification email resent!'))
+  }
 
-    <Button
-      size='lg'
-      variant='primary'
-      onClick={_handleClick}
-    >
-      My email is verified!
-    </Button>
-  </div>
-)
+  _handleClick = () => window.location.reload()
 
-export default VerifyEmailScreen
+  render () {
+    return (
+      <div>
+        <h1>Verify your email address</h1>
+        <p className='mt-5'>
+          Please check your email inbox and follow the link in the message
+          to verify your email address before finishing your account setup.
+        </p>
+        <p>
+          Once you're verified, click the button below to continue.
+        </p>
+        <div className='mt-5'>
+          <Button
+            size='lg'
+            variant='primary'
+            onClick={this._handleClick}
+          >
+            My email is verified!
+          </Button>
+        </div>
+        <div className='mt-5'>
```

Comment:
Use `mt-3` to make spacing will look better:

![image](https://user-images.githubusercontent.com/56846598/91867107-3dad0500-ec41-11ea-8e56-5a5fa304b17a.png)

Reply:
Let's compromise at `mt-4`.